xla/service/cpu/onednn_memory_util.cc

```diff
@@ -45,9 +45,10 @@ namespace cpu {
 struct MemrefInfoPOD {
   int64_t dtype;
   int64_t rank;
+  void* data;
```

Comment:
What cache line size are we aligning for? Also, is there a reason for moving `data` up? (We can still pad when it's the last member.)

Reply:
Aligning for 64 bytes, will add a comment. 
Here's my thought process on moving `data` up:
oneDNN restricts tensor dimensions to a maximum of 12, though in practice we rarely use dimensions larger than 3 or 4.  By moving `data` up, I think we can reduce the total number of cacheline loads from 3 to 2 as dtype, rank, data and dimensions can fit within a single cacheline, while strides can fit within a second cacheline.
Please let me know if this makes sense or if there is something that I missed.